packages/compiler/src/init/scaffold.ts

```diff
@@ -191,7 +191,7 @@ async function writeMain(host: CompilerHost, config: ScaffoldingConfig) {
   const lines = [...config.libraries.map((x) => `import "${x.name}";`), ""];
   const content = lines.join("\n");
 
-  return host.writeFile(joinPaths(config.directory, "main.tsp"), await formatTypeSpec(content));
+  return host.writeFile(joinPaths(config.directory, "main.tsp"), content);
```

Comment:
Remove the format which seems not necessary considering there is only 'import' statements in content and will be high cost to refactor for bundling. please help to double check whether it's fine. thx.